packages/allure-cucumberjs/README.md

```diff
@@ -11,23 +11,33 @@ export default class Reporter extends CucumberJSAllureFormatter {
   constructor(options) {
     super(
       options,
-      new AllureRuntime({ resultsDir: "./out/allure-results" }),
+      new AllureRuntime({ resultsDir: "./allure-results" }),
       {
         labels: {
-          issue: [/@bug_(.*)/],
-          epic: [/@feature:(.*)/]
-        }
+          epic: [/@feature:(.*)/],
+          issue: [/@issue:(.*)/],
+          tms: [/@tms:(.*)/],
+        },
+        issuesTrackerUrlPattern: "http://localhost:8080/issue/%s",
```

Comment:
allure 2 supports different link types. So can we make it like for labels, where you have links configuration section with link type as key and pattern as value? 

```json
links: {
   issue: "http://localhost:8080/issue/%s",
   tms: "http://localhost:8080/tms/%s",
   custom: "http://localhost:8080/custom/%s"
}
```